.eslintrc.json

```diff
@@ -0,0 +1,19 @@
+{
+    "env": {
+        "browser": true,
+        "es6": true
+    },
+    "extends": [
+        "airbnb-base"
+    ],
+    "globals": {
+        "Atomics": "readonly",
+        "SharedArrayBuffer": "readonly"
+    },
+    "parserOptions": {
+        "ecmaVersion": 2018,
+        "sourceType": "module"
+    },
+    "rules": {
+    }
+}
```

Comment:
- I'm gonna have to ask you to please configure this file as the one [right here](https://github.com/microverseinc/linters-config/blob/master/javascript/.eslintrc.json); make it look like this 
![image](https://user-images.githubusercontent.com/46115725/73075285-33dc4b80-3e81-11ea-8a25-70af53a4f69e.png); lint, if necessary and you are golden!

Reply:
Thanks @jofer86!  I thought I already did that, I'll look into it